.eslintrc

```diff
@@ -10,15 +10,20 @@
     "react/prefer-es6-class": 0,
     "react/prefer-stateless-function": 0,
     "react/no-multi-comp": 0,
-    "array-callback-return": "off", 
+    "array-callback-return": "off",
     "arrow-body-style": ["warn", "as-needed"],
-    "comma-dangle": ["warn", "never"],
+    "comma-dangle": ["warn", "always"],
     "import/extensions": "off",
-    "import/no-unresolved": "off",
+    "import/no-unresolved": 1,
+    "indent": ["warn", 2],
     "react/jsx-no-bind": "off",
-    "max-len": "off",
+    "max-len": 1,
```

Comment:
Although, shouldn't it be `"ignoreComments": true`, if you want to ignore comments?